properties.go

```diff
@@ -94,6 +94,15 @@ const (
 	// multicast on startup.
 	EnvVarMulticastEnabled = "SMUDGE_MULTICAST_ENABLED"
 
+	// EnvVarMulticastAnnounceInterval is the name of the environment variable
+	// that describes whether Smudge will attempt to re-announce its presence
+	// via multicast every X seconds.
+	EnvVarMulticastAnnounceInterval = "SMUDGE_MULTICAST_ANNOUNCE_INTERVAL"
```

Comment:
Can this be changed to `EnvVarMulticastAnnounceIntervalSeconds` to agree with `DefaultMulticastAnnounceIntervalSeconds`?